config/routes.rb

```diff
@@ -188,7 +188,9 @@
         resource :hesa_reinstatements, only: :show, path: "/reinstate"
       end
 
-      resource :placements, only: %i[new create edit update]
+      resources :placements, only: %i[new create destroy] do
+        get "destroy", on: :member, to: "placements#confirm_destroy", as: "destroy"
```

Comment:
I think it should be more like delete then destroy

https://github.com/DFE-Digital/register-trainee-teachers/blob/35538b37ae4608f9eb08b6c45df24bb381e84e8d/config/routes.rb#L94